library/proc_macro/src/lib.rs

```diff
@@ -1333,7 +1379,47 @@ impl Literal {
     // was 'c' or whether it was '\u{63}'.
     #[unstable(feature = "proc_macro_span", issue = "54725")]
     pub fn subspan<R: RangeBounds<usize>>(&self, range: R) -> Option<Span> {
-        self.0.subspan(range.start_bound().cloned(), range.end_bound().cloned()).map(Span)
+        bridge::client::FreeFunctions::literal_subspan(
+            self.0.clone(),
+            range.start_bound().cloned(),
+            range.end_bound().cloned(),
+        )
+        .map(Span)
+    }
+
+    fn with_symbol_suffix<R>(&self, f: impl FnOnce(&str, &str) -> R) -> R {
+        self.0.symbol.with(|symbol| match self.0.suffix {
+            Some(suffix) => suffix.with(|suffix| f(symbol, suffix)),
+            None => f(symbol, ""),
+        })
+    }
+
+    fn stringify_impl<R>(&self, f: impl FnOnce(&[&str]) -> R) -> R {
```

Comment:
```suggestion
    fn with_stringify_parts<R>(&self, f: impl FnOnce(&[&str]) -> R) -> R {
```
(and/or add a comment explaining the callback and that this is like this for performance reasons)